.gitignore

```diff
@@ -149,3 +149,4 @@ untitled*
 .mypy_cache
 node_modules
 package-lock.json
+monkeytype.sqlite3
```

Comment:
This is a file that MonkeyType creates to store a database of types, for example when running pytest.